spec/rubocop/cop/style/comment_annotation_spec.rb

```diff
@@ -113,12 +113,27 @@
     RUBY
   end
 
-  it 'accepts a keyword that is somewhere in a sentence' do
-    expect_no_offenses(<<-RUBY.strip_indent)
-      # Example: There are three reviews, with ranks 1, 2, and 3. A new
-      # review is saved with rank 2. The two reviews that originally had
-      # ranks 2 and 3 will have their ranks increased to 3 and 4.
-    RUBY
+  context 'comments in consecutive lines' do
+    it 'accepts a keyword that is somewhere in a sentence' do
+      expect_no_offenses(<<-RUBY.strip_indent)
+        # Example: There are three reviews, with ranks 1, 2, and 3. A new
+        # review is saved with rank 2. The two reviews that originally had
+        # ranks 2 and 3 will have their ranks increased to 3 and 4.
+      RUBY
+    end
+
+    it 'adds an offense for each appended comment' do
```

Comment:
I wonder if appended comment is the best way to name this. To me it sounds confusing.

Reply:
How about trailing comment?

Reply:
I think in most of our code we call those "inline comments". You can grep the code to check.